programs/stake/Cargo.toml

```diff
@@ -13,13 +13,22 @@ edition = { workspace = true }
 bincode = { workspace = true }
 log = { workspace = true }
 solana-account = { workspace = true }
+solana-bincode = { workspace = true }
+solana-clock = { workspace = true }
 solana-config-program = { workspace = true }
 solana-feature-set = { workspace = true }
 solana-instruction = { workspace = true }
 solana-log-collector = { workspace = true }
+solana-native-token = { workspace = true }
+solana-packet = { workspace = true }
+solana-program = { workspace = true }
 solana-program-runtime = { workspace = true }
 solana-pubkey = { workspace = true }
+solana-rent = { workspace = true }
 solana-sdk = { workspace = true }
```

Comment:
Will `sdk` be removed when genesis_config is evicted?

Reply:
Yes